rusty_socket_server/src/connection.rs

```diff
@@ -0,0 +1,55 @@
+use std::net::TcpStream;
+use std::io::{Read, Write};
+use std::sync::{Arc, Mutex};
+use rusty_socket_core::{DataFrame, OpCode};
+
+pub struct Connection {}
+
+impl Connection {
+    pub fn handle_frames(mut stream: TcpStream, active_conn: Arc<Mutex<Vec<TcpStream>>>) {
```

Comment:
Better to use hash map data structure for multicasting messages to group of clients. But this things should be deferred to the application implementing this protocol.

Reply:
I have no idea haha, we can talk about it after you are back.